content/en/workshop/index.adoc

```diff
@@ -6,69 +6,105 @@ images: []
 contributors: []
 ---
 
-= First Workshop of SIGTURK, the ACL Special Interest Group on Turkic Languages
+= The First Annual Meeting of the Special Interest Group on Turkic Languages (SIGTURK) August 15-16 2024, Bangkok (Co-located with ACL)
 
 == Introduction
 
-Welcome to the first edition of the SIGTURK Workshop, as part of ACL 2024, focusing on the computational linguistics of Turkic languages. This workshop aims to bring together researchers and practitioners in the field to discuss, share, and advance the state-of-the-art in this subject of interest.
+We present the first edition of the SIGTURK Workshop representing the annual meeting of the ACL Special Interest Group on Turkic Languages, which aims to provide a new venue that will promote studies on computational linguistics in Turkic languages. Our main objective is to bring together an interdisciplinary community of researchers working on different aspects of natural language processing (NLP) models and corpora in Turkic languages, providing the recently growing number of researchers working on the topic with a means of communication and an opportunity to present their work and exchange ideas. The main objectives of the workshop are:
+To construct and present a wide array of methodology and know-how, including foundational work for formulation and analysis of NLP models suitable for Turkic language processing;
+* To provide a better understanding on how Turkic language typology may impact the applicability of current methods and motivate the development of novel methods that are more generic or competitive in Turkic languages;
+* Preparation of new corpora and resources in under-studied Turkic languages;
+* To build infrastructure by means of software libraries and benchmarks for developing NLP models that would accelerate progress in the field;
+* Promote collaborations across scientists from across the world that have specific interests in the study of computational linguistics in Turkic languages.
+
+By supporting collaborations across research groups working on machine learning, Turkic linguistics, and real-life applications of NLP tasks in various languages, the ultimate goal of the workshop is to enlarge connections across our community and allow rapid development of NLP methods and tools that are applicable to a wide range of Turkic languages.
 
 == Important Dates
 
 [options="header"]
 |===
 | Event | Date
-| Workshop Paper Submission Deadline | TBA
-| Notification of Acceptance | TBA
-| Camera-ready Papers Due | TBA
-| Workshop Dates | TBA
-| Main Conference Dates | TBA
+| First call for papers | February 7, 2024 (Wednesday)
+| Secodn call for papers | March 4, 2024 (Monday)
```

Comment:
Should be `Second` instead of `Secodn`

Reply:
Fixed in 47eb14abba9bb161feaa17f5fc850387f7540629